### ISSUE TITLE
create documentation for nvramcui

### ISSUE THREAD

**Issue by merge** [merge/skulls#60]
Since we include the nvramcui payload (developed as part of the coreboot repo) for doing NVRAM settings, we should be able to point to some docs. It would help to have a list of available settings, see https://github.com/merge/skulls/issues/57

https://www.coreboot.org/Payloads#nvramcui seems outdated and the wiki is being retired anyways.

We could write an nvramcui page for https://doc.coreboot.org/lib/payloads/index.html and put that up for review to coreboot; and then link to that.

**Comment by pgeorgi** [comment]
Good idea, just noting that (the recently added) https://doc.coreboot.org/payloads.html may be a better location for describing the payload
